gno.land/cmd/gnoland/start.go

```diff
@@ -210,8 +247,10 @@ func makeGenesisDoc(
 	}
 
 	// Load distribution.
-	balances := loadGenesisBalances(genesisBalancesFile)
-	// debug: for _, balance := range balances { fmt.Println(balance) }
+	balances, balancesErr := loadGenesisBalances(genesisBalancesFile)
```

Comment:
```suggestion
	balances, err := loadGenesisBalances(genesisBalancesFile)
```

why not this?

Reply:
Updated:

[89dc54f](https://github.com/gnolang/gno/pull/1240/commits/89dc54f8e50db9ac1a17f4897199956a1a8eafc8)